lib/helper.js

```diff
@@ -146,7 +151,29 @@ const getIfExists = get => {
         retryInterval,
         retryTimeout,
       );
-      return await get(tag);
+      const nodeIds = await get(tag);
+      function getText() {
+        return this.innerText;
+      }
+      let elementWrappers = nodeIds.map(nodeId => {
+        if (nodeId instanceof Object) {
+          return Object.assign(nodeId, { description: description });
+        }
+        let basicElementWrapper = {
+          get: async () => nodeId,
```

Comment:
this can be a simple value instead of a func as it returns just the nodeId

Reply:
done